extensions/attention-stream/package.json

```diff
@@ -12,13 +14,13 @@
     "build:glean:metrics": "glean translate ./metrics.yaml ./pings.yaml -f typescript -o src/generated",
     "build:glean:docs": "glean translate ./metrics.yaml ./pings.yaml -f markdown -o docs",
     "lint:glean": "glean glinter ./metrics.yaml ./pings.yaml",
-    "build:developer": "cp manifest.dev.json manifest.json && rollup -c --config-enable-developer-mode && npm run tailwind",
-    "build:emulator": "cp manifest.prod.json manifest.json && rollup -c --config-enable-emulator-mode && npm run tailwind",
+    "build:developer": "rollup -c --config-enable-developer-mode && npm run tailwind",
+    "build:emulator": "rollup -c --config-enable-emulator-mode && npm run tailwind",
```

Comment:
same question as above; shouldn't this have `npm run config:chrome` ?

Reply:
I'm not actually sure we're using the emulator target, but I can. I guess the problem is if you want to run these targets with Firefox you'll need to edit package.json right? Really we should make it configurable, I'm not sure we should do that now though.